backends/qualcomm/passes/fuse_consecutive_transpose.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Qualcomm Innovation Center, Inc.
+# All rights reserved
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import torch
+from executorch.backends.qualcomm.passes.layout_transform import LayoutTransform
+
+from executorch.exir.dialects._ops import ops as exir_ops
+from executorch.exir.pass_base import ExportPass, PassResult
+from executorch.exir.passes import dead_code_elimination_pass
+
+
+class FuseConsecutiveTranspose(ExportPass):
+    """
+    This pass fuses consecutive transpose / permute into one to reduce runtime
```

Comment:
I notice that the `view_copy` node before/after linear stays there, is there any specific reason we keep them?

Reply:
I think we need it because keep_dims is not supported in Qnn Htp.